docs/developing-contracts/examples/DAO.md

```diff
@@ -0,0 +1,196 @@
+---
+sidebar_label: 'DAO'
+sidebar_position: 3
+---
+
+# Decentralized autonomous organization
+
+The smart-contract is avaialbe on GitHub: https://github.com/gear-tech/apps/tree/master/dao
+
+### Structs
+
+```rust
+/// Initial parameters:
+struct InitConfig {
+    admin: H256,
+    approved_token_program_id: H256,
+    guild_bank_id: H256,
+    period_duration: u64,
+    voting_period_length: u64,
+    grace_period_length: u64,
+    dilution_bound: u128,
+    abort_window: u64,
+}
+```
+
+`admin` - the only member that is initially in DAO. He has 1 share that allows him to submit new proposal and therefore add new members.
+
+`approved_token_program_id` - the reference to the token contract (ERC20) that users use as pledge to get the share in the DAO.
+
+`period_duration` - the smallest unit time interval for the DAO.
+
+`voting_period_length` - voting time interval.
```

Comment:
How about `Number of intervals for voting time`? Seems voting duration is `period_duration` * `voting_period_length`.